features/savedata.yml

```
@@ -0,0 +1,6 @@
+name: Save-Data
+description: When a client enables a data-saving mode, the browser sends a `Save-Data` request header which should be used as a signal to reduce data sent to the client. If enabled, websites are encouraged to deliver alternative content, such as smaller image and video resources, or different markup and styling.
```

I think this is the first (primarily) HTTP feature I've reviewed. I'm guessing we'll have to start establishing some conventions here. Here's my take on this (not definitive):

```suggestion
description: Browsers send the `Save-Data` HTTP request header when the user turns on a data-saving mode. It is a hint to reduce data sent to the browser. Servers should respond with alternative content, such as smaller images and videos, or different markup and styling.
```

Some usual stuff here, like putting the entrypoint (the header name in this case) close to the beginning.

But I guess we'll need a pattern for request headers. Something like "Browsers send [`header-name`] when [condition]. Severs should respond with [action]"?

We also need to think about some terminology. For instance, do want to pair up "server" with "browser"? Or do we want to use "server" and "client" (more HTTP-ish terminology)?

Also, for hints, I think it's nice to use the term "hint" specifically, which spares us from having to say "should" or "may" too much.

---

> We could try to establish "browser" for now, use "client" if it seems necessary, and avoid "user agent"?

Yes, this works for me. 👍